eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/EventHubClientImpl.java

```diff
@@ -73,6 +81,38 @@ public EventHubClient apply(MessagingFactory factory) {
                 }, executor);
     }
 
+    public static CompletableFuture<EventHubClient> create(
```

Comment:
>public static CompletableFuture<EventHubClient> create( [](start = 4, length = 55)

Do we need to make this public since we removed Create API on the client?

Reply:
EventHubClient and EventHubClientImpl are in separate packages. If the creates on Impl are not public, then EventHubClient can't call them.